src/utils/data_loaders.py

```diff
@@ -26,12 +26,12 @@ def load_span_data(tokenizer_name, file_name, label_fn=None, has_labels=True):
     [Mr, ., Por, ter, is, nice ]. If the original span indices was [0,2], under the new tokenization, 
     it becomes [0, 3].
     The task file should of be of the following form: 
-        text: str, 
+        text: str
```

Comment:
Strictly speaking, this is not something I should touch, since it's someone else's  function. But the format checker complained, and the following comma is inconsistent with everywhere else, it is not too terrible to make it cleaner, right?

Reply:
Fine by me!